From 54300793675bd45f038a2e342bce966be3fb3686 Mon Sep 17 00:00:00 2001
From: Flominator <github.flominatorgd@gmail.com>
Date: Mon, 26 Oct 2026 20:02:46 +0000
Subject: [PATCH] Updater: Default to Exynos7870-labs OTA channel

OPTIONAL fork-only change. Builds that set lineage.updater.uri (a3y17lte does, in its product makefile) do not need this: Utils.getServerURL() prefers that property over the updater_server_url resource.

Keeping the default in sync anyway makes the fork behave correctly for devices that forget the property, and points the changelog menu at the GitHub Releases that actually host these unofficial builds instead of a non-existent official device page.

Signed-off-by: FlominatorGD <github.flominatorgd@gmail.com>
---
 app/src/main/res/values/strings.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/app/src/main/res/values/strings.xml b/app/src/main/res/values/strings.xml
index 023d0b3..e715c4b 100644
--- a/app/src/main/res/values/strings.xml
+++ b/app/src/main/res/values/strings.xml
@@ -32,7 +32,7 @@
           {type} - Build type
           {incr} - Incremental version
     -->
-    <string name="updater_server_url" translatable="false">https://download.lineageos.org/api/v1/{device}/{type}/{incr}</string>
+    <string name="updater_server_url" translatable="false">https://raw.githubusercontent.com/Exynos7870-labs/lineage_release/lineage-22.2/{device}.json</string>
 
     <string name="verification_failed_notification">Verification failed</string>
     <string name="verifying_download_notification">Verifying update</string>
@@ -84,7 +84,7 @@
     <string name="menu_copy_url">Copy URL</string>
     <string name="menu_export_update">Export update</string>
     <string name="menu_show_changelog">Show changelog</string>
-    <string name="menu_changelog_url" translatable="false">https://download.lineageos.org/<xliff:g id="device_name">%1$s</xliff:g>/changes</string>
+    <string name="menu_changelog_url" translatable="false">https://github.com/Exynos7870-labs/lineage_release/releases</string>
     <string name="menu_ab_perf_mode">Prioritize update process</string>
     <string name="menu_update_recovery">Update recovery</string>
     <string name="toast_forced_update_recovery">It is impossible to disable Lineage Recovery updates on this device.</string>
-- 
2.47.3

